README.md

```diff
@@ -1,3 +1,191 @@
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Copy [link] (https://github.com/unorjikingsley/react-bookstore.git)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+- Open terminal or command prompt and run git clone https://github.com/unorjikingsley/react-bookstore.git
+
+### Install
+
+Install this project with:
+
+- cd react-bookstore
+- run `npm i`
+
+### Usage
+
+To run the project, execute the following command:
+
+- Run `npm run start`
+
+<!-- ### Run tests
+
+To run tests, run the following command: -->
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+<!-- ### Deployment
+
+You can deploy this project using: -->
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Unorji Kingsley**
+
+- GitHub: [Unorji Kingsley](https://github.com/unorjikingsley)
+- LinkedIn: [Unorji Kingsley](www.linkedin.com/in/unorjikingsley)
+
+<!-- 👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle) -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+ ## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[UI]**
+- [ ] **[Functionality]** -->
+- [ ] **[State Management tools]** 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/unorjikingsley/react-mathmagician-calculator/issues)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project, please give it a star. Also, it would be great to suggest ways to make it better and I will definetly look into it.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for this opportunity
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+<!-- ## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+ -->
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](https://github.com/unorjikingsley/React-Bookstore/blob/linters-features/LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>

```

Comment:
- [ ] Kindly ensure that you `MIT` link is not broken. The MIT license gives users :person_curly_hair:  express permission to reuse code for any purpose ☺️.